src/Collector/AssetCollector.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Debug\Collector;
+
+use JetBrains\PhpStorm\ArrayShape;
+use Yiisoft\Assets\AssetBundle;
+
+final class AssetCollector implements CollectorInterface, IndexCollectorInterface
+{
+    use CollectorTrait;
+
+    private array $assetBundles = [];
+
+    public function getCollected(): array
+    {
+        return $this->assetBundles;
+    }
+
+    public function collect(AssetBundle $assetBundle): void
+    {
+        if (!$this->isActive()) {
+            return;
+        }
+
+        $this->assetBundles[] = $assetBundle;
+    }
+
+    #[ArrayShape(['totalAssetBundles' => 'int'])]
+    public function getIndexData(): array
+    {
+        return [
+            'totalAssetBundles' => count($this->assetBundles),
```

Comment:
Better to store int instead of collection of objects if you want to just count it.
WDYT?

Reply:
We need objects as well.